tests/test_dcgm_monitor.py

```diff
@@ -68,6 +69,31 @@ def test_record_memory(self):
         with self.assertRaises(TritonModelAnalyzerException):
             DCGMMonitor(frequency, tags)
 
+    def test_record_utilization(self):
+        # One measurement every 0.01 seconds
+        frequency = 0.01
+        monitoring_time = 10
+        tags = [GPUUtilization]
+        dcgm_monitor = DCGMMonitor(frequency, tags)
```

Comment:
Is this testing a Mocked version of DCGM? I don't see a check to for an actual value. A test like this should take a known input data with a predictable output, and then make sure that output is correct.

Reply:
Can I fix this after #27? This version works with the Mocked version of the DCGM but it is a bit hard to add it here since it depends on #27. I will create another PR to check the actual values. This is not using the mocked version but it works with the Mocked version too.

Reply:
👍